### PR TITLE
Added Countries (Fixed, Updated)

### PR DESCRIPTION
**Pull request by klorpa**
Also added and alphabetized names and nicknames.

* Andorra (Andorran)
* Belize (Belizean)
* Bosnia and Herzegovina (Bosnian)
* Bulgaria (Bulgarian)
* Costa Rica (Costa Rican)
* Croatia (Croatian)
* El Salvador (Salvadoran)
* Federated States of Micronesia (Micronesian)
* Grenada (Grenadian)
* Honduras (Honduran)
* Kiribati (I-Kiribati)
* Kosovo (Kosovan)
* Laos (Laotian)
* Luxembourg (Luxembourgian)
* Macedonia (Macedonian)
* Madagascar (Malagasy)
* Maldives (Maldivian)
* Malta (Maltese)
* Moldova (Moldovan)
* Mongolia (Mongolian)
* Nauru (Nauruan)
* Nicaragua (Nicaraguan)
* Niger (Nigerien)
* Panama (Panamanian)
* Taiwan (Taiwanese)

### CODE REVIEW

**Comment by pregAnon** [comment]
A question: Belarus, Dominican Republic and Scotland seem familiar to me. You wouldn't have happened to have taken them from somewhere now, would you have? A 100% match in nicknames, largescale overlap in names and how you've grouped the trio together leaves me rather suspicious of their origins.

File: src/uncategorized/nationalityToAccent.tw [hunk 1]
```diff
@@ -464,7 +464,37 @@
...
+	<<set $activeSlave.accent = $seed>>
+<<elseif ($activeSlave.nationality == "Mongolian")>>
+	<<set $activeSlave.accent = $seed>>
+<<elseif ($activeSlave.nationality == "Taiwanese")>>
+	<<if $language == "Chinese">>
+	<<set $activeSlave.accent = 0>>
+	<<else>>
+	<<set $activeSlave.accent = $seed>>
+	<</if>>
+<<elseif ($activeSlave.nationality == "Belizean">>
```
Comment (Folant) [inline src/uncategorized/nationalityToAccent.tw]: `)` missing.

**Comment by freecitiesdev** [comment]
klorpa, do you have a response to pregAnon? I'd like to merge this, but I'd also like to have that concern addressed first. If there's any attribution question, drop those countries out of this PR so I can merge the ones that aren't questionable.

**Comment by megas4o** [comment]
Some of your slurs are not correct/rather silly.
https://en.wikipedia.org/wiki/List_of_ethnic_slurs_by_ethnicity
https://en.wikipedia.org/wiki/List_of_ethnic_slurs
http://www.rsdb.org/
https://en.wikipedia.org/wiki/List_of_religious_slurs
Do some research.

**Comment by freecitiesdev** [comment]
Please feel free to continue improving these.

**Pull request merged**